<!DOCTYPE html>
<html class="no-touch">

<head>
    <title>iGEM Calgary // Projects</title>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <link href="bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen">
    <link href="css/global.css" rel="stylesheet" media="screen">
    <link href="css/global.mobile.css" rel="stylesheet" media="only screen and (max-width:600px)">

    <link href='https://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
    <link href="css/projects.css" rel="stylesheet" media="screen">
    <link href="css/projects.mobile.css" rel="stylesheet" media="only screen and (max-width:600px)">
</head>

<body>
    <div id="guide">&nbsp;</div>

    <div w3-include-html="header.html"></div>

    <div id="content">
        <div class="container">
            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2017.png')">
                        <div class="cover"></div>
                        <p class="year">2019</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>iGEM Calgary 2019 project</h2>
                    </div>

                    <div class="full-description">
                        <a href="" style="font-style: italic; font-size: 2em;">2019 Team Wiki</a>
                        <h3>Problem</h3>
                        <p></p>

                        <h3>Solution</h3>
                        <p></p>

                        <h3>Results</h3>
                        <p></p>
                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2017.png')">
                        <div class="cover"></div>
                        <p class="year">2018</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>iGEM Calgary 2018 project</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2018.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2018 Team Wiki</a>
                        <h3>Problem</h3>
                        <p></p>

                        <h3>Solution</h3>
                        <p></p>

                        <h3>Results</h3>
                        <p></p>
                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2017.png')">
                        <div class="cover"></div>
                        <p class="year">2017</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>iGEM Calgary 2017 project</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2017.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2017 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Space agencies and private companies aim to send humans to Mars as early as 2030s. However, many challenges need to be addressed before pioneering Mars. Waste management is a major challenge for future Mars missions as a crew of 6 astronauts will generate about 6 tonnes of solid organic waste. Furthermore, transporting materials to Mars will be challenging due to high transportation costs and difficulty in anticipating every little thing astronauts might need over a two year mission. Consequently, recovering useful resources from waste will be crucial on future long-duration space missions to minimize waste and transportation costs.</p>
                        
                        <h3>Solution</h3>
                        <p>The 2017 team developed a start-to-finish process to produce a bioplastic on Mars from solid human waste using genetically engineering bacteria. The produced bioplastic can be used to 3D print useful items for astronauts. We consulted with astronauts, engineers, and scientists at the Canadian Space Agency, NASA, and German Aerospace Center to understand the needs and the design constraints. Economic feasibility of the system was evaluated using NASA's Equivalent System Mass analysis, where emphasis was placed on minimizing the system's mass, volume, and power requirements. This solution provides a feasible way to manufacture items on Mars, while minimizing waste.</p>
                        
                        <h3>Results</h3>
                        <p>Our team received a gold medal at the International Genetically Engineered Machine (iGEM) competition in Boston. Our project was also nominated for Best Manufacturing Project. After the competition, a part of the project was tested in microgravity as part of Canadian Reduced Gravity Design Challenge (CAN-RGX) organized by SEDS-Canada in collaboration with the National Research Council of Canada and the Canadian Space Agency.</p>
                        
                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2014.png')">
                        <div class="cover"></div>
                        <p class="year">2014</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>iGEM Calgary 2014 project</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2014.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2014 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Infectious diseases such as typhoid fever, meningitis, pneumonia and visceral leishmaniasis have similar symptoms as malaria and are often misdiagnosed as malaria in resource-poor developing countries lacking suitable medical diagnostic facilities. Failure to properly identify such diseases prevents medical professionals from administering appropriate treatments in a timely manner, which results in economic costs and human suffering. Furthermore, patients who test negative for malaria but show its clinical signs and symptoms are often given antimalarial drugs despite their diagnosis. The over-prescription of antimalarials results in the emergence of drug resistance.</p>
                        <h3>Solution</h3>
                        <p>The 2014 team developed a novel, genome-based, rapid point-of-care synthetic biological device to simultaneously diagnose multiple infectious diseases prevalent in developing countries. We engineered <i>Bacillus subtilis</i> to generate chromophoric reporter proteins in response to pathogenic genetic markers. To understand the needs and design constrains for a device in developing counties, we consulted with experts including academics, physicians in developing counties, and organizations such as Foundation for Innovative New Diagnostics (FIND). Based on their feedback, we developed a portable, user-friendly, economically feasible device and built a prototype of the device.</p>
                        <h3>Results</h3>
                        <p>Our team received a gold medal at the International Genetically Engineered Machine (iGEM) competition in Boston. The team was also given a unique opportunity to attend the Biological and Toxin Weapons Convention at the United Nations in Geneva, Switzerland.</p>
                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2013.png')">
                        <div class="cover"></div>
                        <p class="year">2013</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>Rapid detection of <em>E. coli</em> in cattle</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2013.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2013 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Contamination of beef with pathogenic <em>E. coli</em> O157:H7 has shaken public perception of meat, costing the industry millions of dollars with product recalls. Pathogenic <em>E. coli</em> inhabit the guts of healthy cattle. Moreover, a small percentage of these cattle are known as "super shedders" which pass huge quantities of these bacteria in their faeces. Shedding from these animals causes increased colonization of neighbouring cattle. Increased levels of <em>E. coli</em> in cattle populations from "super shedders" increase the likelihood that these bacteria will be transferred from a cow's hide or digestive system to consumer meats during slaughtering.</p>

                        <h3>Solution</h3>
                        <p>The 2013 team built a proof-of-concept sensor so that stakeholders in the beef industry can identify cattle shedding abnormally high amounts of <em>E. coli</em> O157:H7. We worked closely with ranchers, feedlot owners, slaughterers, and academics, realizing that this industry can lower <em>E. coli</em> in populations by separating high shedders from normal animals. We want to lessen pathogenic <em>E. coli</em> in cattle populations to reduce opportunities for beef contamination in slaughter plants.</p>
                        <p>We applied protein engineering, nanotechnology, and material science to construct a cost-effective, rapid-acting, portable DNA sensor resembling a home pregnancy test. This work is the start of a system which could be accessible to workers outside of laboratories, who will be able to use fecal samples to determine the extent of <em>E. coli</em>-shedding beef cattle.</p>

                        <h3>Results</h3>
                        <p>In 2013, we were first-runner-up among 64 teams at the North America regional competition at the University of Toronto and were awarded Best Team Wiki. In the global finals at MIT, we won Best Food and Energy Project among the undergraduate teams.</p>
                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2012.png')">
                        <div class="cover"></div>
                        <p class="year">2012</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>Detect and destroy: Building FRED and OSCAR</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2012.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2012 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Development of the Alberta oil sands contaminates water used in the extraction process with various toxins. These toxic and corrosive compounds are an environmental and economic concern to Alberta and Canada.</p>

                        <h3>Solution</h3>
                        <p>We constructed two systems to tackle this problem which we called FRED and OSCAR. Check out the team Wiki for more details on this project. FRED, the Function and Robust Electrochemical Detector, was expanded from our 2011 project. In FRED, an engineered bacterium responds to tailings ponds toxins, and these changes are detected using a portable electronic device. We improved the FRED bacterium to respond to a broader variety of toxins and continued development of the device.</p>
                        <p>OSCAR, the Optimized System for Carboxylic Acid Remediation, is a system for converting these toxins into usable fuels. We developed multiple bacteria to tackle a broad group of toxins. These components were integrated into a physical bioreactor as part of a proposed system for eventual industry deployment.</p>
                        <p>Through the development of FRED and OSCAR, we consulted industry to tailor our system to their needs. From these discussions, we integrated control measures--a kill switch--into our system to address concerns of release of engineered organisms into the environment.</p>

                        <h3>Results</h3>
                        <p>In 2012, we enjoyed our most successful year to date. At the American West regional competition, we won five of eight possible awards. We won the Best Human Practices award in the global finals at MIT, where we finished amongst the top 16 teams worldwide.</p>

                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2011.png')">
                        <div class="cover"></div>
                        <p class="year">2011</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>A naphthenic acid biosensor for tailings pond remediation</h2>
                        <!--<p>Here is my single-line summation of what the project was.</p>-->
                    </div>

                    <div class="full-description">
                        <a href="http://2011.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2011 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Naphthenic acids (NA) are naturally-occurring components of crude oils, including the bitumen in Alberta's oil sands region. During separation of bitumen from the sands, alkali-soluble NA remain in the waste stream that is deposited into tailings ponds to allow for solids consolidation and water recycling. Naphthenic acids typically persist in oil sands tailings ponds and have known toxic and corrosive effects that impact water disposal and reuse, and pond reclamation efforts. It is thus critical to have sensitive methods for NA measurement in active ponds and those undergoing reclamation. Existing NA detection methods using organic and analytical chemistry tools are expensive, time consuming, and require high technical expertise.</p>

                        <h3>Solution</h3>
                        <p>The University of Calgary iGEM team developed synthetic biology tools to develop a low cost, sensitive, and selective biosensor that can detect naphthenic acids in oil sands tailings ponds and other environments. Biosensors provide an alternate, low-cost and easy method to monitor for NA in tailings ponds and other environments.</p>
                        <p>Our team designed a naphthenic acid sensing organism using a novel screening method for proteins which could bind NA. Additionally, we utilized bioinformatics in order to identify candidate genes which could also be responsible for naphthenic acid detection. In order to provide a sensitive, quantitative, output for our sensor, we designed a way to report the level of these toxins electrochemically. Enzymes that the cell produce can cleave compounds to make them electroactive and this can be detected using a potentiostat.</p>

                        <h3>Results</h3>
                        <p>Our team successfully identified a gene element capable of responding to naphthenic acids. In addition, we demonstrated that it is possible to sense these compounds electrochemically. Our group designed a prototype of our device and were very successful at the competition, winning Best Wiki and Best Experimental Measurement and Best Environmental Project overall. Our group was active in participating in our community, working with establishing an iGEM group at the University, designing a music video, and outreaching to high school and middle school students.</p>

                    </div>
                </div>
            </div>

            <div class="row project">
                <div class="project-thumbnail-container">
                    <div class="project-thumbnail" style="background-image: url('images/project_thumbnail_2010.png')">
                        <div class="cover"></div>
                        <p class="year">2010</p>
                    </div>
                </div>

                <div class="project-description">
                    <div class="synopsis">
                        <h2>Translating stress into success: A protein expression toolkit</h2>
                    </div>

                    <div class="full-description">
                        <a href="http://2010.igem.org/Team:Calgary" style="font-style: italic; font-size: 2em;">2010 Team Wiki</a>
                        <h3>Problem</h3>
                        <p>Many projects in synthetic biology necessitate over expression of recombinant proteins in microorganisms. A major stumbling block however, is often an inability to express functional protein. This situation is difficult to manage and troubleshoot as it is often unclear why expression is failing. Problems can occur in transcription, translation or folding of the protein.</p>

                        <h3>Solution</h3>
                        <p>The University of Calgary iGEM team has designed a system that can accurately and visually report whether a gene is being transcribed and/or translated successfully. Using a system of stress-responsive promoters, we also designed and tested reporter constructs capable of detecting protein misfolding events in the periplasm or cytoplasm. Building on this, we also designed genetic components to fine tune expression levels of a given protein so as to optimize production, increasing the likelihood of obtaining functional protein. To further understand protein misfolding we also built an equation-based, multivariant model of inclusion body formation. Inclusion bodies are aggregates of protein that can occur when proteins misfold. Finally, we used a series of podcasts to explore the social implications of our project in the context of the growing synthetic biology field.</p>

                        <h3>Results</h3>
                        <p>Our team successfully designed and tested reporter constructs for misfolding events in both the cytoplasm and the periplasm and were able to use them to troubleshoot another teamâ€™s project. In addition, we completed our inclusion body formation model and a series of podcatsts and blogs exploring ethical concerns surrounding our project as well as a series of other outreach activities. For our accomplishments, our team was awarded a gold medal at the World jamboree.</p>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div w3-include-html="footer.html"></div>

    <script src="js/load.js"></script>

    <!-- JavaScript plugins (requires jQuery) -->
    <script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="bootstrap/js/bootstrap.min.js"></script>
    <!-- Optionally enable responsive features in IE8 -->
    <script src="bootstrap/js/respond.min.js"></script>

    <script src="js/global.js"></script>
    <!--<script src="/js/guide.js"></script>-->
    <script src="js/projects.js"></script>
</body>
</html>
